study.md

```diff
@@ -27,17 +28,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+HTML & View Source.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Kinda because the DOM is a representation of your HTML resource.
 ```
 
 In your own words, why do you think the DOM is important?
 
 ```md
-<!-- your answer here -->
+The DOM is important because if there are mistakes in your HTML the DOM will fix the browser for you.
 ```
```

Comment:
We definitely want to try to avoid mistakes in our HTML. While the DOM may sometimes may display things as we intended them to look we can't count on this. The real power of the DOM is that it allows our web pages to be interactive!